SpatialGDK/Extras/schema/component_presence.schema

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+// The ComponentPresence component should be present on all entities that 
+// should be load-balanced. This contains a single property which is the 
+// list of component IDs that are present on the entity. Currently, this 
```

Comment:
nit: This is a different definition to the one mentioned in the component presence class

Reply:
actually this one is correct